.gitignore

```diff
@@ -26,7 +26,7 @@ build/Release
 node_modules
 
 # Babel compiled sources
-lib
+#lib
```

Comment:
Sorry about that, didn't mean to include that as part of the pull request. 

I was attempting to make sure post build and pull down from git that things worked as expected (since I don't have an npm account to publish to and didn't want to publish a fork of this project on npm)

I'll revert this change.